### PR TITLE
Issue#1043 movie

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Create MovieDocument

Contributes to #1043

### CODE REVIEW

File: libs/movie/src/lib/movie/+state/movie.firestore.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import { DateRangeRaw } from "@blockframes/utils/date-range";
+import { firestore } from "firebase";
```
Comment (ioaNikas) [inline libs/movie/src/lib/movie/+state/movie.firestore.ts]: You can go deeper in the import with "firebase/app".

File: apps/backend-functions/src/data/types.ts [hunk 2]
```diff
@@ -148,7 +141,7 @@ export interface Notification extends BaseNotification {
 }
 
 export interface SnapObject {
```
Comment (phpgeekfr) [inline apps/backend-functions/src/data/types.ts]: Can you add a comment to explain what this object is about?

File: libs/movie/src/lib/movie/guards/movie-organization-active.guard.ts [hunk 3]
```diff
@@ -5,12 +5,6 @@ import { Router } from '@angular/router';
 
 export const movieOrganizationActiveQuery = (id: string): Query<Movie> => ({
   path: `movies/${id}`,
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/guards/movie-organization-active.guard.ts]: you removed
 ```  stakeholders: (movie: Movie) => ({
    path: `movies/${movie.id}/stakeholders`,
    organization: stakeholder => ({
      path: `orgs/${stakeholder.id}`
    })
  })
```

Is it intentional ? If yes, maybe you need to rename `movieOrganizationActiveQuery` ?

Comment (Hugo-Hontans) [reply libs/movie/src/lib/movie/guards/movie-organization-active.guard.ts]: Sure I rename it

File: apps/backend-functions/src/data/types.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/backend-functions/src/data/types.ts]: We will do a point to data/types.ts in other PR

**Pull request merged**